app/cosi/main.go

```diff
@@ -0,0 +1,253 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"golang.org/x/net/context"
+	"io"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"bytes"
+	"github.com/dedis/cothority/app"
+	"github.com/dedis/cothority/lib/cosi"
+	"github.com/dedis/cothority/lib/dbg"
+	"github.com/dedis/cothority/lib/network"
+	"github.com/dedis/cothority/lib/sda"
+	"github.com/dedis/crypto/config"
+)
+
+func printUsageAndExit(msg string) {
+	if msg != "" {
+		fmt.Fprintf(os.Stderr, "%s", msg)
+	}
+
+	fmt.Fprintf(os.Stderr, `Usage:
+First make sure that you have a valid group toml file. Either create a
+local one by running several instances of cosid (follow the instructions when
+running cosid the first time) or use a group toml of an existing CoSi group.
+
+To collectively sign a text message run:
+	cosi -m “<Message to be signed>” -c <cosi-group.toml>
+If you would instead like to sign a message contained in a file you can the following command:
+	cosi -f <file-to-be-signed> -c <cosi-group.toml>
+It will create a file file-to-be-signed.sig containing the hash of the file and the signature.
+
+To verify the signature on a file add the verify flag (-v):
+	cosi -f <file-to-be-signed> -c <cosi-group.toml> -v
+This command opens the corresnponding .sig file and validates the contained signature.
+
+Example usuage (create a file my-local-group.toml using cosid first):
+cosi -f my-file.txt -c my-local-group.toml`)
+	os.Exit(1)
+}
+
+var f *flag.FlagSet
+var m *flag.FlagSet
+
+func init() {
+	f = flag.NewFlagSet("f", flag.ContinueOnError)
+	m = flag.NewFlagSet("m", flag.ContinueOnError)
+	//dbg.SetDebugVisible(3)
```

Comment:
Remove - if you merge #301 there will be a `dbg.AddFlags()` method to automatically add the `-debug`, `-debug-color` and `-debug-time` flags. It even evaluates environment-variables.